pkgs/by-name/in/instant-scala/package.nix

```diff
@@ -0,0 +1,35 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  scala-cli,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "instant-scala";
+  version = "0.1.0";
+
+  src = fetchFromGitHub {
+    owner = "jatcwang";
+    repo = "instant-scala";
+    rev = "refs/tags/v${finalAttrs.version}";
+    sha256 = "jqSvKTL8NzqjwqDj/+55YWecx2bnzuArP8RdfH5q/1U=";
+  };
+
+  buildInputs = [
+    scala-cli
```

Comment:
You also need to patch the use of scala-cli in the script, otherwise it is not picked up.